books/RayTracingInOneWeekend.html

```diff
@@ -523,17 +511,7 @@
 </div>
 
 <div class='together'>
-The rules of vector algebra are all that we would want here, and if we expand that equation and
-move all the terms to the left hand side we get:
-
-  $$ t^2 \vec{\mathbf{b}}\cdot\vec{\mathbf{b}}
-     + 2t \vec{\mathbf{b}} \cdot \vec{(\mathbf{a}-\mathbf{c})}
-     + \vec{(\mathbf{a}-\mathbf{c})} \cdot \vec{(\mathbf{a}-\mathbf{c})} - R^2 = 0
-  $$
-</div>
-
-<div class='together'>
-The vectors and $R$ in that equation are all constant and known. The unknown is $t$, and the
+The vectors and $R$ are constant and known. The unknown is $t$, and the
```

Comment:
In general, the original text sets up the reader for understanding the mathematics behind ray intersection with a surface defined by an implicit equation. In this situation it picks the simple case of ray-sphere intersection, but provides a basis by which a reader could also derive the math for ellipsoids, cones, torii and other surfaces. Removing the derivation leaves the reader with only the final equations for a sphere.

I believe the text is weakened by removing these explanations.

Reply:
sure, okay.  i'm not a math whiz, especially with 3d